lib/Position/Base91ParserUtils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Telemetry = require('./../MessageModels/Telemetry.js');
+
+module.exports = {
+    parseBase91Telemetry: (bodyContent) => {
```

Comment:
Nit, I think we can improve var naming across this function a little bit

- bodyContent -> comment
- msgString -> telemetryMatch

Reply:
Agreed